src/ucp/core/ucp_request.h

```diff
@@ -110,7 +110,10 @@ struct ucp_request {
          * operations */
         struct {
             ucp_ep_h              ep;
-            void                  *buffer;  /* Send buffer */
+            union {
+                void              *buffer;    /* Send buffer */
+                void              *flush_ctx; /* flush context */
```

Comment:
if ep_flush was needed for ucp_atomic_post before this pr, why was flush_ctx not needed before?

Reply:
user has request headroom for this purpose